protoc-gen-twirp_ruby/main.go

```diff
@@ -68,7 +96,14 @@ func (g *generator) generateRubyCode(file *descriptor.FileDescriptorProto, pbFil
 
 	indent := indentation(0)
 	pkgName := file.GetPackage()
-	modules := splitRubyConstants(pkgName)
+
+	var modules []string
+	if file.Options.RubyPackage != nil {
```

Comment:
If I omit `options ruby_package = "some_package";` in the `.proto` file I get a `panic: runtime error: invalid memory address or nil pointer dereference` on this line, which I am guessing is because `file.Options` might be nil.

Changing this line to `if file.Options != nil && file.Options.RubyPackage != nil` seemed to fix it.